compiler/tools/fpp-to-dict/test/top/multipleTops.fpp

```diff
@@ -132,6 +132,27 @@ module Module1 {
       severity activity high \
       format "Arg1 is {f}"
 
+    @ Event with throttle
+    @ Multiple lines of annotation
+    @ And not used on purpose
```

Comment:
Do we need these annotation lines? It looks like they were copy-pasted from an existing event.

Reply:
I agree, looks like copy and paste and not needed!